### PR TITLE
Jumanah Alhareth

### PR DESCRIPTION
**Pull request by jumanah18**

### CODE REVIEW

File: Jumanah Alhareth
Comment (Radwan-Albahrani) [inline Jumanah Alhareth:1]: what is this file??

File: weatherapp.dart [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+class MyWeatherApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Weather App',
+      home: WeatherPage(),
+    );
+  }
+}
```
Comment (Radwan-Albahrani) [inline weatherapp.dart:13]: this should all be in main.dart

File: weatherapp.dart [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                print('Refreshing weather...');
+              },
+              child: Text('Refresh Weather'),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
```
Comment (Radwan-Albahrani) [inline weatherapp.dart:64]: design doesnt meet requirements

**Review by Radwan-Albahrani — COMMENTED** [review]
While this is a good attempt, the design doesnt meet the requirements of this task. Code separation and custom widgets were a requirement of this task. Excellent work though, keep it up

**Comment by jumanah18** [comment]
Hello Radwan,
As l previously said I really could not get the work done because of issues
l had while running the code (gets choppy, slow etc), but l tried to do
what l could get done with the task so hopefully l did not mess it up.
Regarding the file l accidentally made a new pull request on my account and
uploaded the file there but forgot to delete it😅. Pardon me

**Comment by Radwan-Albahrani** [comment]
Hello hello!!

I've discovered a solution that should hopefully work on your laptop. 

Don't run the flutter app on an emulator. Run it as a web app then resize the chrome window to look like a phone device. It should cause less overheating issues.

**Comment by jumanah18** [comment]
That's smart! Will try that thank you🙏🏼

**Pull request closed without merging**